curvefs/src/client/fuse_volume_client.cpp

```diff
@@ -277,18 +234,37 @@ CURVEFS_ERROR FuseVolumeClient::FuseOpCreate(fuse_req_t req, fuse_ino_t parent,
 CURVEFS_ERROR FuseVolumeClient::FuseOpMkNod(fuse_req_t req, fuse_ino_t parent,
                                             const char *name, mode_t mode,
                                             dev_t rdev, fuse_entry_param *e) {
-    LOG(INFO) << "FuseOpMkNod, parent: " << parent
-              << ", name: " << name
-              << ", mode: " << mode
-              << ", rdev: " << rdev;
-    return MakeNode(req, parent, name, mode, FsFileType::TYPE_FILE, rdev, e);
+    VLOG(3) << "FuseOpMkNod, parent: " << parent << ", name: " << name
+            << ", mode: " << mode << ", rdev: " << rdev;
+    return MakeNode(req, parent, name, mode, FsFileType::TYPE_VOLUME, rdev, e);
 }
 
 CURVEFS_ERROR FuseVolumeClient::FuseOpFsync(fuse_req_t req, fuse_ino_t ino,
```

Comment:
FuseOpFlush do not implement in line275
<img width="830" alt="image" src="https://user-images.githubusercontent.com/15689619/161885593-0e7c3f0a-03d6-4011-be67-416abcd658f5.png">

Reply:
fix